src/commands/run-tests-solidus-older.yml

```diff
@@ -9,17 +9,17 @@ parameters:
 steps:
   - test-branch:
       branch: v4.3
-      rails_version: "~> 7.1"
+      rails_version: ['>= 7.0', '< 7.2']
```

Comment:
```suggestion
      rails_version: ['>= 7.1', '< 7.2']
```

Reply:
It seems, we need a string

```
rails_version: "'~> 7.1.0"
```

Reply:
Unfortunately, the Orb needs the requirement to be a string.